
string rigid_body_name
geometry_msgs/Pose pose
geometry_msgs/Pose rot